packages/mst-middlewares/test/custom.ts

```diff
@@ -0,0 +1,137 @@
+import { test } from "ava"
+import atomic from "../src/atomic"
+import { types, addMiddleware, getSnapshot } from "mobx-state-tree"
+
+let error: any = null
+
+function omitNextAbort(call, next) {
+    // omit next() / abort()
+}
+function nextAndAbort(call, next, abort) {
+    abort("someValue")
```

Comment:
I think we should clearly document that `abort` and `next` should be invoked _synchronously_